.github/workflows/ci.yml

```
@@ -199,6 +199,11 @@ jobs:
             coverage.filtered.info
             coverage-summary.txt
           retention-days: 7
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4
+        with:
```

actionlint (1.7.12)

[error] "with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)